Lmod-illumina.spec

```diff
@@ -49,12 +49,16 @@ BuildRequires:  lua-filesystem
 BuildRequires:  lua-json
 BuildRequires:  lua-posix
 BuildRequires:  lua-term
+%if %{rhel6_based}
+BuildRequires:  lua-bit32
+Requires:  lua-bit32
+%endif
 Requires:       lua
 Requires:       lua-filesystem
 Requires:       lua-json
 Requires:       lua-posix
 Requires:       lua-term
-Requires:       cm-config-cm = %cmrelease
+###Requires:       cm-config-cm = %cmrelease  ### unhash me when all is fine
```

Comment:
@johnnydevaprasad :
The above line could rather be enclosed in `%if` & `%endif` directives, to make the spec file generic. 
Any ideas what that condition should look like?

otherwise this PR is getting ready for merging, please review!